config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   root "welcome#index"
+
+  resources :foods
```

Comment:
Would have loved to see an `only: [index]` here so we aren't creating extra routes.